src/main/kotlin/org/rust/ide/core/overrideImplement/generateUtil.kt

```diff
@@ -0,0 +1,112 @@
+package org.rust.ide.core.overrideImplement
```

Comment:
I was comparing with Kotlin plugin where all kotlin files (not classes) start with a lowercase letter.